CONTRIBUTING.md

```diff
@@ -95,11 +95,14 @@ it will create a new folder inside `packages` with a simple boilerplate for your
 - The version number is calculated by a combination of the output of `git describe` and the `publish:prerelease` script in the root package.json. Specifically having `major` is appropriate for the pre-7.0 development phase, but we should change it to `minor` after 7.0.0 is finally released.
 
 ### Release
-- If necessary, make and merge a PR to contain any last minute changelog entries or other housekeeping items
-- Make a PR that contains the file changes from `pnpm lerna version --no-push 7.0.0` (update the version as necessary)
+- Review code commits and decide what type of release this will be - PATCH, MINOR, MAJOR
+- If necessary, make and merge a PR to contain any last minute housekeeping items (docs, tests, etc.)
+- Make a PR that contains the file changes from `pnpm lerna version --no-push 7.0.0` (update this version number as appropriate)
```

Comment:
@mfedderly what do you do exactly to "Make a PR" that contains the file changes from the version command?  Do you do this using Github CI? Through the Github web UI?

And how exactly do you "push the release PR" in the follow on step below?

Reply:
When you run that lerna command it changes the package.json files to update all of the versions everywhere. You just add all the changes and commit it.

Reply:
Thanks @mfedderly, the part I didn’t understand is how/where the PR should get created if the next step is to “push the release PR”.  I’m not sure what it means to push a PR. Can you provide more detail on what you type for each step and where you are doing it? (git client, github website).

Reply:
ah ok so in the console its something like this:

```
git fetch origin
get checkout origin/master -b mf/release-7.0.0
pnpm lerna version --no-push 7.0.0
git add packages/*/package.json (or something like this? check what files got changed)
git commit -m "Release v7.0.0"
git push origin mf/release-7.0.0 --follow-tags
# use the link in the output from ^ to create a PR and get it merged in the normal fashion
```